Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,532 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function (_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        "Tween",
+        _("Tween animation"),
+        _("Animate object properties via tweening."),
+        "Matthias Meike, Florian Rival",
+        "Open source (MIT License)"
+      )
+      .setExtensionHelpPath("/all-features/tween");
+
+    var tweenBehavior = new gd.BehaviorJsImplementation();
+
+    tweenBehavior.updateProperty = function (
+      behaviorContent,
+      propertyName,
+      newValue
+    ) {
+      return false;
+    };
+
+    tweenBehavior.getProperties = function (behaviorContent) {
+      var behaviorProperties = new gd.MapStringPropertyDescriptor();
+      return behaviorProperties;
+    };
+
+    tweenBehavior.setRawJSONContent(
+      JSON.stringify({})
+    );
+
+    const behavior = extension
+      .addBehavior(
+        "TweenBehavior",
+        _("Tween animation"),
+        "TweenBehavior",
```

Comment:
For the default name, let's use "Tween" (I know the example use "DummyBehavior" but it's a mistake I think on my side). Adding Behavior in the name of the behavior does not give more information and make things more verbose when writing expressions:

`MyObject.TweenBehavior::MyExpression();` vs `MyObject.Tween::MyExpression();`

Second will be more readable :)